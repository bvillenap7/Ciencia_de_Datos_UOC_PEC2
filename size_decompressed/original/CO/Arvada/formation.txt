Looking west over the Rockies from my Arvada back yard, i saw 3 white lights flickering in a row. It occurred to me that from my angle;2011-04-17 21:00:00;15 mins;Looking west over the Rockies from my Arvada back yard, i saw 3 white lights flickering in a row.  It occurred to me that from my angle they could have been in a triangle formation.  I thought i may have been mistaking them for Orion's Belt, but they did not coincide with the constellation's location in the sky...AND they were gone ten minutes later.   I frequently observe the night sky from my backyard, and haven't seen this phenomenon before or since.;39.8289982352941;-105.110888235294
8 objects flying completely silent with three lights each in the night sky;2015-10-16 13:15:00;20 seconds;8 objects flying completely silent with three lights each in the night sky Hard to tell what it was.  Looked potentially military but was oddly completely silent.   Looked like 8 craft with three lights, each flying in very close V-formation and seeming to move slightly independently, but in a very very tight formation that looked odd, and almost too close and right to be planes, all moving at the same speed.  Each craft had three lights In a triangle shape.;39.8289982352941;-105.110888235294
7-9 glowing white lights in arrow formation flying east to west fast and low;2016-09-30 20:30:00;30 seconds;7-9 glowing white lights in arrow formation flying east to west fast and low Looking east I noticed a white glowing object traveling low and fast coming towards me as it got closer I noticed there were more lights with it around 7-9.they were in a arrow formation.  My first thought was, okay it's a comet breaking up but as it flew over my head I noticed they were all the exact same size and glowing the same white cream luminescent color and in almost perfect formation..there was no sound at all.whats crazy is I can stare east and see planes from a distance coming from DIA all the time but this just appeared.It was like you couldn't see them from afar until it got close to you.I star gaze all the time when I'm outside smoking a cigarette (bad habit), so I always see falling stars (comets) and single flying lights (satellites) but this was unexplainable.thats why I had to document it here.   P.S as soon as it was over my head I ran 10ft to the back door to yell at my wife to come see this and instead of her running straight out the back door f! rom the couch she paused to turn the thermostat off for the swamp cooler and it emitted sparks( something it never done before) by the time she ran out it was to late because my backyard neighbor has a big tree and it was out of view headed for the mountains. ((NUFORC Note:  Witness elects to remain totally anonymous provides no contact information.  PD));39.8289982352941;-105.110888235294
We drive our car in the night when we see objects of orange and red lights in a formation in the sky . couple they move and the others;2017-02-01 20:00:00;15 minutes;We drive our car in the night when we see objects of orange and red lights in a formation in the sky.  Couple they move and the others stayed there.;39.8289982352941;-105.110888235294
I was standing outside facing W from approx. 64th and llamar.  I like to star gaze.  Out of seemingly nowhere.;2017-12-09 19:30:00;20 minutes;I was standing outside facing west from approximately 64th and Llamar.  I like to star gaze.  Out of seemingly nowhere a long line of blinking red and green lights just appeared in the sky, no big flash or start, it was very subtle. they were definitly lights though.  I thought they might be crane lights but there were thousands of them in a straight line that seemed to stretch for miles.   Maybe they were lights on an airplane, but they moved across the sky so slowly that you'd think they weren't even moving but over the course of a hour they slowly drifted westward.  A plane would've flown much faster across the sky at that distance relative to myself.  Best estimate of distance is at the start being maybe a half mile to a mile above me and a few miles to the south west.   There didn't seem to be any other lights.  It didn't do anything but rapidly blink between red and green and slowly drift west.;39.8289982352941;-105.110888235294
Very long, perfect string of red lights evenly spaced with no sound moving slowly from east to west;2017-12-09 21:15:00;10 minutes;Very long, perfect string of red lights evenly spaced with no sound moving slowly from east to west At approx. 21:00 hrs through 21:15 hrs my mom was taking the dog out and noticed in the south eastern sky traveling to the south western sky, where Orion would be later in the night, a string of at least 12 visible lights that were red occasionally blinking white (one at a time but not in order) then going back to red.  The distance was unknown but there was no sound to be associated with the lights.   She came and got me about five minutes after she first noticed the lights.  They were evenly spaced and appeared to be moving slowly from east to west finally disappearing over the mountains.  I was able to grab image stabilizing binoculars which just made the red and white lights appear larger.  A minute passed and I grabbed my dad from inside and all three of us watched as the lights slowly moved over the mountains and disappeared.  At the end of the formation to the east there was another set of lights which looked like a normal plane which appeared to be following the red and white light formation.  The plane was lower in the sky to the other lights and moving at a different speed and angle and had normal blinking running lights including green.  What really stood out was the exact distance of the lights to one another, the fact they appeared to be in a perfect line with no sound and the slow rate of speed.  It was impossible to tell wether it was one huge craft or many smaller crafts.   The following day on the news reported it was a military formation of C 130 (I think), performing training HALO mission flying to Nevada.;39.8289982352941;-105.110888235294
34 Points of light traveling in a straight line equidistant from each other over the course of 8 minutes.;2020-02-27 05:00:00;8 minutes;34 Points of light traveling in a straight line equidistant from each other over the course of 8 minutes. On the morning of February 27th I was leaving for work at 5 am.  I looked up and saw what I first thought was a satellite moving from the northwest to the southeast.  But then noticed another moving in the same path behind the first some distance back, then another behind that moving in the same path, then another.  I began counting how many I saw continue in this fashion until I reached 34 total, all following in a straight line on the exact same path approximately equidistant from each other.  Of the 34 I counted all but about 4 followed the same path.  The other 4 were moving in the same general direction but on a different path a ways outside of the main path.  Also they all seemed to be traveling slightly faster than you normally see satellites moving.  I stood there watching this event for somewhere between 5 and 10 minutes.  I witnessed this from my home in Arvada.;39.8289982352941;-105.110888235294
