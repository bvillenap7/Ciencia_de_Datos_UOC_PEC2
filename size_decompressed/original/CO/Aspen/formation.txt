3 orbs in sky remote Colorado oct 12 2013  moving in up and down to side to side plus sign formations .;2013-10-12 22:00:00;15-30 seconds;3 orbs in sky remote colorado oct 12 2013  moving in up and down to side to side plus sign formations . I was driving from utah to colorado springs to visit my wife oct 12 2013 . Like an hour into colorado a little after 10 pm.  I'm  not exactly certain exact city I was in but it  remote colorado like 1 hour west of aspen.  Anyways I was driving and I looked at the sky and saw 3 orbs moving in the craziest formations up and  down and side to side simultaniously when I looked up like 3 seconds after one of the orbs bursted into space with a red trail behind it and like 10 seconds later the other two orbs disipated.  I pulled out my blackberry to try and get a pic but sighting only lasted like 10 seconds didn't have enough time driving and part of me was just entranced cuz I couldn't believe what I was actually seeing.  I've never seen anything move that fast and in formations that don't match any aircraft I know of.  I'm a skeptic so if anyone has a good explanation please let me know cuz I'm honestly baffeled;39.1977833333333;-106.825375
