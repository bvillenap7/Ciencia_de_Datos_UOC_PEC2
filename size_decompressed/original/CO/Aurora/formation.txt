6 crescent shapes in the sky southeast of denver;2001-10-15 17:30:00;3-5min;6 crescent shapes in the sky southeast of denver on the way home from my job myself and two friends were driving southeast on interstate 225 along side of the Cherry Creek Dam. just as we rounded the turn and could see the open sky in front of us i noticed what looked like about 6 very large white crescent shapes in the sky.  they were not moving at all but the phenomenon lasted a few minutes, the distance that we were away from them it looked as though they could have benn as tall as a five story building and spaced the same distance between eachother.  as we drove towards the shapes more the crescent on the far righthand side began to shift and appeared to dissolve a little at the bottom and just a quickly as they had appeared they just vanished. ((NUFORC Note:  Date is approximate.  PD));39.6695444272445;-104.803991486068
Group of faint red lights flying in a boomerang formation very high and quiet;2001-11-18 04:30:00;2 minutes;Group of faint red lights flying in a boomerang formation very high and quiet While looking at the Leonid meteor shower, I noticed a satellite traversing the sky directly overhead, that satellite was moving in a NNE direction. While looking at the satellite I saw a group of faint red lights moving in a boomerang formation from North to South that intersected the satellite path. The red lights clearly passed in front of the satellite. I did not file this report until I reasoned everything I saw that night and I went to the Satellite Prediction Report that I recieved from http://liftoff.msfc.nasa.gov/RealTime/JPass/PassGenerator/....the satellite I may have seen was proabably ALOUETTE 1 or RadarSat......but I know for sure that the red lights I saw were not a plane or satellite. I asked my next door neighbor who was outside also with his Celestron telescope 'did you see that' and he said 'yes I see it too...your are not crazy'.;39.6695444272445;-104.803991486068
White dots over Aurora.;2014-07-04 22:30:00;10 minutes;white dots over aurora. I seen 3 bright lights in formation of a triangle slowly hovering. I was looking toward the sky watching fireworks while walking when all the sudden down the path I was walking.   I seen these 3 bright white lights not flashing just hovering for a approximately 3 minutes.  Then like Christmas lights they disappeared then randomly changing formation.  Then a few minutes later they follows in a line of 8 lights that appeared head west one synchronized in flight pattern looked the shape of an upside down hook in the sky.;39.6695444272445;-104.803991486068
Broad daylight, blue sky, 7 silvery things that looked like they were trying to form a circle.  ((anonymous report));2018-08-28 11:00:00;<1 minute;Broad daylight, blue sky, 7 silvery things that looked like they were trying to form a circle. Broad daylight, looked up at the clear blue sky and saw this cluster of 7 silvery, shiny like misshapen circles that had a thin dark border around each one.  These 7 circles were in kind of a messed up connected circle, with one of them seeming to be lagging behind.  They would be there one second and - poof! - would just totally be gone.  In a second or two, you could see them again.   When they would come back into view the big circle of them together would be a little more of an actual circle.  The lagging one seemed to finally catch up with the rest and they were shiny silver, and when the lagger was almost in place the whole thing just disappeared totally and we didn't see it again, watched for a long time and it never reappeared.   It was totally silent, was going North to South, seemed to be moving smoothly although in a disorganized circle at first.  It was at a higher elevation than the airplanes that we always see coming and going from the airport.   It left us just stupefied, and asking each other over and over, Did you SEE that? ((NUFORC Note:  Witness elects to remain totally anonymous provides no contact information.  PD));39.6695444272445;-104.803991486068
Objects were flying in a straight line in the distance, disappearing above one by one;2020-01-14 19:00:00;5 minutes;Objects were flying in a straight line in the distance, disappearing above one by one Objects looking like stars were flying in a straight line in the distance.  They all traveled the same speed with the same distance to each other and getting a little brighter at first when they getting closer, disappearing above one by one by one.;39.6695444272445;-104.803991486068
3 objects in a triangle formation;2021-01-31 11:00:00;5 minutes;3 objects in a triangle formation I was driving south on I-225 around the 6th Ave exit around 11-11:30 am on Sunday 1/31/21.  When I looked to my left and noticed 3 objects flying in a triangle formation.  They were moving very slow and had a very strange contrail that moved around them almost like it would be a high speed re-entry type.  This was not like a normal aircraft where the contrail comes out of the back of the engine on each wing or the tail of the aircraft.  I watch it for at least 5 minutes until I drove past it, if I was not driving I could have watched it for a lot longer.  I am an A&P mechanic that works for a major airline and the reason I am reporting this, is because this was 100% not normal.;39.6695444272445;-104.803991486068
