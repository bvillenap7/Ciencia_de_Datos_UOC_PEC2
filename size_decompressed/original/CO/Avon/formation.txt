~15 Vertical flashing red lights headed west in Colorado.;2017-12-09 21:15:00;8+ minutes;~15 Vertical flashing red lights headed west in Colorado. There were approximately 15 flashing red lights in a vertical formation.  They were seen headed NW, and then shifted to W.  The lights were flashing in sync, and appeared to have an effect of “falling” - as the lights Forbes farther in the distance over the mountain, the blinking appeared randomized.  When we first saw it, it looked like a cell phone tower, because off how straight the lights were after it was in the distance, it looked more like a grouping, in no particular order.;39.6313;-106.5108
~15 Vertical flashing red lights headed west in Colorado;2017-12-09 21:15:00;8+ minutes;~15 Vertical flashing red lights headed west in Colorado There were approximately 15 flashing red lights in a vertical formation.  They were seen headed NW, and then shifted to W.  The lights were flashing in sync, and appeared to have an effect of “falling” - as the lights traveled farther in the distance over the mountain, the blinking appeared randomized.  When we first saw it, it looked like a cell phone tower, because off how straight the lights were after it was in the distance, it looked more like a grouping, in no particular order.   There was no detectable sound.   Dogs in the neighborhood were barking loudly.  Usually one barks at a deer or two, but to hear many dogs barking at 9:15pm was odd.;39.6313;-106.5108
