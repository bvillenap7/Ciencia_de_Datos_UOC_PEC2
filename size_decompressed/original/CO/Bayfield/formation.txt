Single file dots of unblinking light. Moved sw to ne.   ((Starlink satellites?));2020-03-05 19:45:00;10 minutes;Single file dots of unblinking light.  Moved sw to ne;37.261;-107.6068
