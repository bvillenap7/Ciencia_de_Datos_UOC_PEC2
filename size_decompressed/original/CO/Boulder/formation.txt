I was sitting in my back yard talking on the phone.When I saw 1 green dot fly south Followed by 2 more green dots flying parallel to ea;1995-11-15 22:30:00;10 sec.;I was sitting in my back yard talking on the phone.When I saw 1 green dot fly south Followed by 2 more green dots flying parallel to each other. All 3 objects were flying in the same heading. 1 followed by 2, 3 sec. later. I was sitting in my back yard witch has a great view of the front range of the Rocky mountians. I was on the phone looking up in the night sky. The moutian range is to the west of me and I am facing north. the first green light was headed in a southernly coarse. I mentioned to the person on the phone that I had seen a shooting star. Three seconds had passed and then two more green dots That were flying directly side by side flew along the same flight pattern . the last two lights flew on either side of the first one . the approximate alltitude was probably 50,000 ft.;40.0217727272726;-105.258365102639
Three black objects flying in close, diamond shaped formation.;1999-11-13 21:00:00;10 seconds;Three black objects flying in close, diamond shaped formation. Two friends and I were sitting in an open space watching for meteors and generally looking at the stars.  One of them pointed to an area of the sky and said What is that?  We all looked and saw what appeared to be 3 smallish objects flying in very tight formation.  The objects emitted no sound at all, and it was difficult to judge the altitude because none of the objects had any lights whatsoever.  Only the light pollution from the city illuminated them.  They flew very quickly from east to west.;40.0217727272726;-105.258365102639
5 glowing golden cigar shaped discs moving in formation;2004-05-12 04:02:00;30 seconds;5 glowing golden cigar shaped discs moving in formation We were out to see the Space Station come over when 5 golden glowing discs, a bit larger than footballs and the same shape, came by us, over the roof, and out to parking lot where they disappeared.  They were moving fast and as if one unit but 5 different discs.  All were glowing gold and moved with no noise..;40.0217727272726;-105.258365102639
Formation of 4. Greenish white Flying V & pyramid shape displayed.;2014-07-04 23:30:00;5-10 minutes;Formation of 4. Greenish white  Flying V & pyramid shape displayed. We were watching the fireworks of course, then between the show in Westminster & Denver 4 greenish white lights appeared.  They circled with each other, dancing, then formed a very articulate V shape pointing to earth, somewhere near Lafayette.  They descended toward the horizon, broke formation & regrouped in a perfect equilateral triangle, with one in the middle.  I assume it was a pyramid in the air.  They descended further toward the ground & then disappeared.;40.0217727272726;-105.258365102639
Husband and myself witnessed a line of star-like lights.  ((NUFORC Note:  Starlink satellites?  PD));2019-11-24 04:40:00;15;At 4:40-458am my husband and myself witnessed a line of star-like lights That seemed to enter our atomosphere and move across the sky traveling south to north east. The lights were in a straight line formation, evenly spaced except every 8-10 lights, two were traveling almost abreast. They were moving at a steady speed. Definitely not aircraft. We thought satellite except for the numbers.  We believe there were 75-100!!  It was strange how the lights entered into our our view. Almost like they flew in through a port hole above Louisville Co. We are educated and not crazy.  This was weird.;40.0217727272726;-105.258365102639
