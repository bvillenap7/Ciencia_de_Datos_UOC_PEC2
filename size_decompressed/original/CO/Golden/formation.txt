Several reflective flying objects, making strange formations w/liquid like motion in broad daylight near Rocky Flats (Have Photos).;2006-04-12 13:51:00;5 mins;Several reflective flying objects, making strange formations w/liquid like motion in broad daylight near Rocky Flats (Have Photos). Was Driving North on Highway 93 coming from Golden heading towards Boulder. About 2 miles S. of Rocky Flats. Saw a group of reflective objects making different formations (very high up) in the sky slightly NW of my vehicle. They were moving in what I can only describe as a liquid type motion. Kind of like mercury. They moved in unison. The group split in half w/one half of the group mirroring the other half's motion. It was very strange, so I pulled over on the side of the highway and tried to get a better look. They were very reflective and all seemed to be identical. The objects were slowly dancing in that liquid motion over head for about 5 Min's. They then made a formation of a triangle (not a V) and jetted off very quickly headed NW (more West than North) and were gone. Anyone else in the area on this date see anything similar? Have digital photos.;39.7598265306123;-105.220002040816
Advanced light patterns turning in sharp angles and moving at high speeds in a fluid manner that no known human aircraft could handle.;2013-07-04 21:40:00;~5 minutes;Advanced light patterns turning in sharp angles and moving at high speeds in a fluid manner that no known human aircraft could handle. A couple friends and I (3 people total) decided to climb up Green Mountain to view all of Denverâ€™s fireworks for the 4th. We could see all of Denver from about Alameda North.   At about 9:40 PM (21:40) we saw 6-8 really bright lights traveling in an oval pattern in the sky.  Using our geographical reference, they looked to be above Arvada or Westminster.  They were moving quicker and more fluid like than any other aircraft I had observed.   The circling oval pattern abruptly stopped after about 10 - 20 seconds.  Now only 4 lights remained and they quickly shuffled around each other in their own unique oval shaped patterns. After about 5 seconds of this the lights reduced down to three in which they formed a triangle in the sky. The lights then began to weave in and out of each other, yet always returning back to the original triangle shape.  The sharp degree turns that these crafts would turn at was remarkable they could flip a direct 180 degree turn at a really high velocity as if momentum was not a factor.   This triangular light pattern lasted for about 20 seconds until 2 suddenly disappeared as if flying north at a really fast speed.   The last light remained for a while longer, the departure of this light was observed as if it were shrinking in size and dimming in luminescence to blend with the city street lights until we could see it no longer.  We potentially considered this last beam of light to have landed as it sunk to blend in with the city lights.   All three individuals can vet for what they saw on the mountain that day, and we all agree that these air crafts were far more advanced than anything we have seen in the skies before.  It was a very inspiring and beautiful spectacle we were graced with.;39.7598265306123;-105.220002040816
Several lights in a V formation flew over head at a steady but slow pace, from N to S.  ((anonymous report));2016-08-11 23:40:00;3 minutes;While watching for meteriods, I watched  several lights in a V formation flew over head at a steady, but slow, pace moved from North to south.  The craft made no sound as it passed by. ((NUFORC Note:  Witness elects to remain totally anonymous provides no contact information.  PD));39.7598265306123;-105.220002040816
